themes-default/slim/views/partials/home/banner.mako

```diff
@@ -152,7 +152,9 @@
                     </td>
                     <td align="center">
                     % if cur_show.network:
-                        <span title="${cur_show.network}" class="hidden-print"><img id="network" class="show-network-image" width="54" height="27" src="images/network/nonetwork.png" lazy="on" series="${cur_show.slug}" asset="network" alt="${cur_show.network}" title="${cur_show.network}" /></span>
+                        <span title="${cur_show.network}" class="hidden-print">
+                            <asset default="images/network/nonetwork.png" series-slug="${cur_show.slug}" type="network" cls="show-network-image" :link="false" width="54" height="27" alt="${cur_show.network}" title="${cur_show.network}"></asset>
```

Comment:
1. No need for `:link="false"` it seems, since it defaults to false. But maybe it's good for readability?
2. I remember having an issue when trying to move the network images to use the asset component.
I think it was the missing error handling when a network logo isn't available.

Reply:
1. Yeah let's leave it for readability.